feature/platform/tests/optics_power_and_bias_current_test/optics_power_and_bias_current_test.go

```diff
@@ -179,28 +166,22 @@ func TestOpticsPowerUpdate(t *testing.T) {
 	}
 }
 
-// findTransceiverName provides name of transciever port corresponding to interface name
+// findTransceiverName provides name of transceiver port corresponding to interface name
 func findTransceiverName(dut *ondatra.DUTDevice, interfaceName string) (string, error) {
-	var (
-		transceiverMap = map[ondatra.Vendor]string{
-			ondatra.ARISTA:  " transceiver",
-			ondatra.CISCO:   "",
-			ondatra.JUNIPER: "",
-			ondatra.NOKIA:   "-transceiver",
-		}
-	)
-	transceiverName := interfaceName
-	name, ok := transceiverMap[dut.Vendor()]
-	if !ok {
-		return "", fmt.Errorf("No transceiver interface available for DUT vendor %v", dut.Vendor())
-	}
-	if name != "" {
+	switch dut.Vendor() {
+	case ondatra.ARISTA:
+		suffix := " transceiver"
 		interfaceSplit := strings.Split(interfaceName, "/")
 		interfaceSplitres := interfaceSplit[:len(interfaceSplit)-1]
-		transceiverName = strings.Join(interfaceSplitres, "/") + name
-		if dut.Vendor() == ondatra.NOKIA {
-			transceiverName = interfaceName + name
-		}
+		return strings.Join(interfaceSplitres, "/") + suffix, nil
+	case ondatra.CISCO:
+		return "", nil
```

Comment:
umm...this changed the original business logic, right? It should return `interfaceName`

Reply:
Yep fixed a few minutes before you left the comment :) sorry about that